site/app/views/forum/ForumThreadView.php

```diff
@@ -917,7 +923,9 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
                     "is_anon" => $first_post["anonymous"],
                     "render_markdown" => $first_post["render_markdown"],
                     "author_info" => $author_info,
-                    "deleted" => $first_post['deleted']
+                    "deleted" => $first_post['deleted'],
+                    "has_attachment" => $first_post['has_attachment'],
+                    "post_attachment" => $post_attachment,
```

Comment:
(Leaving this here since I can only comment on lines which were changed)

Can you remove the extraneous commented-out `var_dump()` below?